etc/org.opencastproject.speechtotext.impl.engine.WhisperCppEngine.cfg

```diff
@@ -1,12 +1,18 @@
+<<<<<<< HEAD
 # Configuration for setting a custom path to the Whisper command line tool
 # Default: whisper
 #whispercpp.root.path=whispercpp
+=======
+# Configuration for setting a custom path to the Whisper.cpp command line tool
+# Default: whisper.cpp
+whispercpp.root.path=/home/lars/dev/whisper.cpp/main
+>>>>>>> f7b28fafd2 (Auto-encode audio track for Whisper)
```

Comment:
Unresolved merge conflict residues.

Reply:
Ups, that's a leftover from some local tests.
Fixed.